src/OpenLoco/Windows/CompanyWindow.cpp

```diff
@@ -2271,16 +2277,87 @@ namespace OpenLoco::Ui::Windows::CompanyWindow
             Common::repositionTabs(self);
         }
 
+        // this will prepare _commonFormatArgs array before drawing the StringIds::challenge_value
+        // after that for example it will draw this string: Achieve a performance index of 10.0% ("Engineer")
+        // 0x004384E9
+        static void sub_4384E9()
```

Comment:
This is a really simple function. I suggest doing this function next. 
For now make it expose the fact that its setting up the common_format_args by returning a FormatArguments.

Reply:
Actually thinking about it more make it take a FormatArguments and then this function will populate those args. It will work like a FormatTo function from OpenRCT2.

Reply:
OK, will do it in new PR.